### PR TITLE
HorizontalRuler component

### PR DESCRIPTION
**Pull request by bernardodiasc**
https://github.com/x-team/auto-ui/issues/21

screen shot 2017-11-13 at 3 24 15 pm

### CODE REVIEW

File: src/components/HorizontalRuler.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import { PureComponent } from 'react'
```
Comment (nicksp) [inline src/components/HorizontalRuler.js]: You forgot to import `React`

Comment (bernardodiasc) [reply src/components/HorizontalRuler.js]: I think the `React` is only required when using JSX syntax. Are you sure we need this @nicksp?

Comment (joshwnj) [reply src/components/HorizontalRuler.js]: That's correct.

**Comment by joshwnj** [comment]
👍

**Pull request merged**